maintainers/maintainer-list.nix

```diff
@@ -8779,6 +8779,12 @@
     email = "kirill.wedens@gmail.com";
     name = "wedens";
   };
+  wesliao = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add wesliao``.